recipes/fisher/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "fisher" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7519d2c884be6f5844bfde900fd294e98a33fe9a943c6b03800513cd281859c3
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - numpy
+    - cython
+    - pytest
```

Comment:
You can drop `pytest` from the `host` section. Also the `setup.py` doesn't use `cython`, so you can probably drop that too. It looks like the package is only built against the generated `.c` file and never actually uses cython